src/platform/graphics/webgpu/webgpu-render-target.js

```diff
@@ -1,6 +1,30 @@
 import { Debug, DebugHelper } from '../../../core/debug.js';
 import { WebgpuDebug } from './webgpu-debug.js';
 
+/**
+ * Private class storing info about color buffer.
+ *
+ * @ignore
+ */
+class ColorAttachment {
+    /**
+     * @type {GPUTextureFormat}
+     * @private
```

Comment:
I don't think you meant to mark this a private.

Reply:
That is the only way to avoid type errors, as WebGPU types cannot be exported. So I added those as workarounds really.

Reply:
Ooooh, OK.